### PR TITLE
O3-3553: Update Lab metadata with Viral Load.

### PR DESCRIPTION
**Pull request by VeronicaMuthee**
This PR adds viral load to basic lab collection.

### CODE REVIEW

**Comment by gracepotma** [comment]
I did this with Veronica; we're just waiting for approval. @ibacher or @denniskigen - is there anything I should do before merging this, which you would normally do?

**Review by ibacher — APPROVED** [review]

**Comment by denniskigen** [comment]
Sorry, just lost track of this

**Comment by VeronicaMuthee** [comment]
@denniskigen no worries, Ian approved. Thanks all.

**Pull request merged**
